integr8ly/application-monitoring-operator
# Intly 5461 add gen csv

### Description:
Added gen command to makefile for generating new csv files

JIRA: https://issues.redhat.com/browse/INTLY-5461

### Verification:
- Clone this repo
- Update `AMO_VERSION` in the makefile to a new version number
- Update `PREV_AMO_VERSION` to the old value for `AMO_VERSION`
- Run `make gen/csv` and verify that a new manifest is generated with the updated version numbers
- Verify the value of `currentCSV` in `application-monitoring-operator-package.yaml` is updated to the newly specified version.
